Velaptor/OpenGL/TextureBatchItem.cs

```diff
@@ -1,62 +1,100 @@
-﻿// <copyright file="TextureBatchItem.cs" company="KinsonDigital">
+// <copyright file="TextureBatchItem.cs" company="KinsonDigital">
 // Copyright (c) KinsonDigital. All rights reserved.
 // </copyright>
 
+namespace Velaptor.OpenGL;
+
 using System;
 using System.Diagnostics.CodeAnalysis;
 using System.Drawing;
 using System.Globalization;
 using System.Text;
-using Velaptor.Graphics;
-
-namespace Velaptor.OpenGL;
+using Graphics;
 
 /// <summary>
 /// A single texture batch item that can be rendered to the screen.
 /// </summary>
-internal struct TextureBatchItem : IEquatable<TextureBatchItem>
+internal readonly struct TextureBatchItem : IEquatable<TextureBatchItem>
 {
     /// <summary>
-    /// The source rectangle inside of the texture to render.
+    /// Initializes a new instance of the <see cref="TextureBatchItem"/> struct.
+    /// </summary>
+    /// <param name="srcRect">The rectangular section inside of a texture to render.</param>
+    /// <param name="destRect">The destination rectangular area of where to render the texture on the screen.</param>
+    /// <param name="size">The size of the rendered texture.</param>
+    /// <param name="angle">The angle of the texture in degrees.</param>
+    /// <param name="tintColor">The color to apply to an entire texture.</param>
+    /// <param name="effects">The type of effects to apply to a texture.</param>
+    /// <param name="viewPortSize">The size of the view port.</param>
```

Comment:
Refactor to say viewport without a space.
```suggestion
    /// <param name="viewPortSize">The size of the viewport.</param>
```